src/main/java/hudson/plugins/sonar/client/SQProjectResolver.java

```diff
@@ -79,6 +80,13 @@ public ProjectInformation resolve(@Nullable String serverUrl, @Nullable String p
 
       return projectInfo;
 
+    } catch (HttpException e) {
+      if (e.code() == 404) {
+        Logger.LOG.log(Level.FINE, "Error fetching project information: {0}", e.getMessage());
```

Comment:
Fine? is this really going to get used?

Reply:
> Fine? is this really going to get used?

The main issue was that logs got polluted by these error messages if the analysis was not found.
This way we can hide the log messages from the logs by default, but allow us to instruct the user to increase the log level in case of issues that would benefit from this 'debug' entry.

Having typed al this, I'm open to just keeping it on 'info' or just dropping it completely.

Reply:
I honestly don't know how easy it is to enable fine, and if that's something often done. I'd assume fine is only used during for debugging while developing. Maybe debug is actually used in production? Really don't know but I agree that not showing in info is probably a good idea.

Reply:
Jenkins has a UI for configuring a logger so this would be 'easy' enough to do, but I'm in the dark if this is something used often or not, it's a guestimate on my end.